sdcm/tester.py

```diff
@@ -335,6 +338,20 @@ def init_nodes(db_cluster):
         if db_cluster.non_seed_nodes:
             db_cluster.wait_for_init(node_list=db_cluster.non_seed_nodes)
 
+    @staticmethod
+    def valid_init_nodes(db_cluster):
+        start_time = time.time()
```

Comment:
Since Python 3.3 it's better to use `time.perf_counter()` instead.

Reply:
changed